### ISSUE TITLE
Update dependencies to latest package versions

### ISSUE THREAD

**Issue by pboychev-bcom** [vmware/build-tools-for-vmware-aria#463]
### Description

Check all TypeScript modules and update the used dependencies to the latest versions of the packages.  
Fix any regressions as dependencies are updated.

**Comment by VenelinBakalov** [comment]
Sounds like a good idea, I am just not sure if this can be covered by one issue. Or maybe we can use 1 issue and open multiple PRs since it sounds like a big topic to go through every single dependency and update to latest. Also, I think there is a chance that by the time we close the last dependency update, the first ones that we closed might already be outdated. But at least they will be with a fairly new version.

**Comment by pboychev-bcom** [comment]
Definitely, updating to latest available through multiple PRs sounds like the way to go :)

**Comment by VenelinBakalov** [comment]
Closed with https://github.com/vmware/build-tools-for-vmware-aria/pull/582
